### PR TITLE
V1.3.x

### PR DESCRIPTION
**Pull request by carl-mastrangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,37 +28,37 @@ Download [the JARs][]. Or for Maven with non-Android, add to your `pom.xml`:
 <dependency>
   <groupId>io.grpc</groupId>
   <artifactId>grpc-netty</artifactId>
-  <version>1.2.0</version>
+  <version>1.3.0</version>
```
Comment (ejona86) [inline README.md]: The README change was supposed to be in a different commit so we could cherry-pick to master. I guess we just won't do that this time.

Comment (carl-mastrangelo) [inline README.md]: I don't mind duplicating this.
